src/features/ITGovernance/Requester/BusinessCase/AlternativeSolution/AlternativeSolutionA.tsx

```diff
@@ -42,135 +41,91 @@ const AlternativeSolutionA = ({
-                      if (
-                        isFinal &&
-                        alternativeSolutionHasFilledFields(values)
-                      ) {
-                        validateForm().then(err => {
-                          if (Object.keys(err).length === 0) {
-                            dispatchSave();
-                            history.push('alternative-solution-b');
-                          } else {
-                            window.scrollTo(0, 0);
-                          }
-                        });
-                      } else {
-                        history.push('alternative-solution-b');
-                      }
-                    }}
-                    base
-                  >
-                    {t('alternativeB')}
-                  </IconButton>
-                </div>
-              )}
             </Form>
 
-            <ButtonGroup>
-              <Button
-                type="button"
-                outline
-                onClick={() => {
-                  dispatchSave();
-                  setErrors({});
-                  const newUrl = 'preferred-solution';
-                  history.push(newUrl);
-                }}
-              >
-                {t('Back')}
-              </Button>
-              <Button
-                type="button"
-                onClick={() => {
-                  dispatchSave();
-                  const newUrl = alternativeSolutionHasFilledFields(
-                    businessCase.alternativeB
-                  )
-                    ? 'alternative-solution-b'
-                    : 'review';
-                  if (isFinal && alternativeSolutionHasFilledFields(values)) {
-                    validateForm().then(err => {
-                      if (Object.keys(err).length === 0) {
-                        history.push(newUrl);
-                      } else {
-                        window.scrollTo(0, 0);
-                      }
-                    });
-                  } else {
-                    history.push(newUrl);
-                  }
-                }}
-              >
-                {t('Next')}
-              </Button>
-            </ButtonGroup>
+            <Button
+              type="button"
+              disabled={!isFormValid}
+              className={classnames('usa-button', {
+                'no-pointer': !isFormValid
+              })}
+              // onClick={() => {
+              //   validateForm().then(err => {
+              //     if (Object.keys(err).length === 0) {
+              //       dispatchSave();
+              //       const newUrl = 'alternative-analysis';
+              //       history.push(newUrl);
+              //     } else {
+              //       window.scrollTo(0, 0);
+              //     }
+              //   });
+              // }}
+
+              // TODO: NJD - I couldn't get valdiation to work properly here - I don't think we need it since we validate on the actual button
```

Comment:
The validation in the commented out `onClick` code above worked for me after removing the disabled button state. Happy to sync on this if you want!

Reply:
Yeah lets sync - did you get it to pass validation?

I was having issues with the validation expecting a nested Yup object and I was passing a bare object representing the individual solutions.